pkg/proxy/policy/policy.go

```diff
@@ -95,7 +101,7 @@ const (
 type URLMatchRule struct {
 	Host      string       `json:"host"`
 	Path      string       `json:"path"`
-	PathMatch MatchingType `json:"matching_type"`
+	PathMatch MatchingType `json:"matchingType"`
```

Comment:
hmm it's not clear what matching this controls. I think the json key should say "path" somehow. Maybe "matchPath" or "matchPathBy" and then "prefix" and "full" would be a bit clearer